omod/pom.xml

```diff
@@ -22,27 +22,28 @@
 		<dependency>
 			<groupId>org.openmrs.api</groupId>
 			<artifactId>openmrs-api</artifactId>
-			<type>jar</type>
+            <scope>provided</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.openmrs.api</groupId>
 			<artifactId>openmrs-api</artifactId>
-			<type>test-jar</type>
+            <scope>provided</scope>
```

Comment:
These api dependencies should not be needed. They are defined in the parent pom.